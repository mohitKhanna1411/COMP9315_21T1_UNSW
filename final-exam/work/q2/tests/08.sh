./bnl 20 10 12
