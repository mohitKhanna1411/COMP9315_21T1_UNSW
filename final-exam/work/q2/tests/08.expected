Running: ./bnl 20 10 12

Frames:     [00] [01] [02] [03] [04] [05] [06] [07] [08] [09] [10] [11]
Contents:    R18  S00  S01  S02  S03  S04  S05  S06  S07  S08  S09  R19
PinCount:      0    0    0    0    0    0    0    0    0    0    0    0
Popularity:    2    3    3    3    3    3    3    3    3    3    3    2
Clock: 0

#requests: 220
#releases: 220
#hits    : 190
#reads   : 30
